Installation/include/CGAL/config.h

```diff
@@ -41,6 +40,11 @@
 #define _SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING 1
 #endif
 
+#ifdef CGAL_USE_SSH
+//to avoid clashing between windows.h and winsock2.h
+#define _WINSOCKAPI_
+#endif
+
```

Comment:
Please move that block inside `#ifdef CGAL_INCLUDE_WINDOWS_DOT_H`, just before the inclusion of `<windows.h>`. The intent will be clearer.

Reply:
I can't, the problem appears even without that specific include. There must be another dependency that includes windows.h somewhere.

Reply:
Then move the hack in `Polyhedron/demo/Polyhedron/config.h`, so that `CGAL/config.h` is not touched. `"config.h"` is supposed to be included first by all files of the demo.